db/seeds.rb

```diff
@@ -33,10 +33,21 @@
 Country.destroy_all
 puts "Cleaning TOPICS database..."
 Topic.destroy_all
+puts "Cleaning OPPORTUNITY_TYPES database..."
+OpportunityType.destroy_all
 puts "Cleaning VOLUNTEERS database..."
 Volunteer.destroy_all
+puts "Cleaning ORGANISATIONS database..."
+Organisation.destroy_all
+puts "Cleaning ORGANISATION_COUNTRIES database..."
+OrganisationCountry.destroy_all
```

Comment:
You should destroy the OrganisationCountry/Topic first, only then the Organisation. You added dependent destroy in the model, so actually you don't need to destroy the OrganisaitonCountry/Topic here at all, they were already destroyed